apps/backoffice-v2/src/common/components/organisms/ImageViewer/ImageViewer.SelectedImage.tsx

```diff
@@ -42,7 +43,9 @@ export const SelectedImage = forwardRef<HTMLImageElement | HTMLIFrameElement, TS
       );
     }
 
-    return (
+    return isPlaceholder ? (
```

Comment:
`isPlaceholder` is meant to be true if an invalid (or no URL) URL is passed, or we are unable to display an image. Skeletons should be visible if either we're fetching the URL for the image or while the asset itself is being downloaded. What's the behaviour for cases with no image?